src/ann_solo/spectral_library.py

```diff
@@ -103,7 +103,8 @@ def __init__(self, filename: str) -> None:
                            self._library_reader.spec_info['charge'].items()
                            if len(charge_info['id']) >= config.num_list]
             for charge in sorted(ann_charges):
-                self._ann_filenames[charge] = f'{base_filename}_{charge}.idxann'
+                self._ann_filenames[charge] = \
+                    f'{base_filename}_{charge}.idxann'
```

Comment:
**nitpick:** I typically prefer using parenthesis instead of backslash for line breaks.
```python
self._ann_filenames[charge] = (
    f'{base_filename}_{charge}.idxann'
)
```

Reply:
Agree. Although eventually I need to use black on the entire project to get a consistent and enforced code style everywhere. I didn't want to do it yet while we're working on this PR to make it easier to see what's changed. We'll use black after this functionality is completed.